### PR TITLE
ci: pin `import_gpg` action to hash

### LINKED ISSUE #3711: CI: pin import_gpg action to hash
At these two locations:

https://github.com/build-trust/ockam/blob/e5dcdecf3f83e417a27f1830a19da8d108c561c8/.github/workflows/release-binaries.yml#L34-L35

https://github.com/build-trust/ockam/blob/e5dcdecf3f83e417a27f1830a19da8d108c561c8/.github/workflows/create-release-pull-request.yml#L61-L62

Change

```
build-trust/.github/actions/import_gpg@custom-actions
```

to:

```
build-trust/.github/actions/import_gpg@a6377d3c2dac878b92a0da26cdf3da2856c64840
```

### PR DESCRIPTION
**Pull request by miles170**
## Current Behavior

`import_gpg` action reference by branch name

## Proposed Changes

`import_gpg` action reference by commit hash

Closes #3711.

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository.

### CODE REVIEW

**Comment by mrinalwadhwa** [comment]
@miles170 The changes look great!!
Thank you for spending time on this, before we can merge please

- [x] Accept the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding your Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository.

The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will make the commit as verified.

**Comment by miles170** [comment]
CLA signed (https://github.com/build-trust/ockam-contributors/commit/c7a2ec80e6535372872a893a30169e7c0dc2e8bc)

File: .github/workflows/create-release-pull-request.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@ jobs:
           echo "sha=$sha" >> $GITHUB_OUTPUT
 
       - name: Import GPG key
-        uses: build-trust/.github/actions/import_gpg@custom-actions
+        uses: build-trust/.github/actions/import_gpg@a6377d3c2dac878b92a0da26cdf3da2856c64840
```
Comment (mrinalwadhwa) [inline .github/workflows/create-release-pull-request.yml:62]: checked https://github.com/build-trust/.github/tree/a6377d3c2dac878b92a0da26cdf3da2856c64840

File: .github/workflows/release-binaries.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ jobs:
           ref: ${{ github.event.inputs.release_branch }}
 
       - name: Import GPG key
-        uses: build-trust/.github/actions/import_gpg@custom-actions
+        uses: build-trust/.github/actions/import_gpg@a6377d3c2dac878b92a0da26cdf3da2856c64840
```
Comment (mrinalwadhwa) [inline .github/workflows/release-binaries.yml:35]: checked https://github.com/build-trust/.github/tree/a6377d3c2dac878b92a0da26cdf3da2856c64840

**Review by mrinalwadhwa — APPROVED** [review]
That looks great. Thank you!

**Comment by mrinalwadhwa** [comment]
@miles170 Thank you for another awesome contribution! 🥳

**Pull request merged**